build/rpms/cri-tools.spec

```diff
@@ -0,0 +1,20 @@
+Name: cri-tools
+Version: OVERRIDE_THIS
+Release: 00
+License: ASL 2.0
+Summary: Container Runtime Interface tools
+
+URL: https://kubernetes.io
+
+%description
+Binaries to interface with the container runtime.
+
+%prep
+tar -xz -f {crictl-v1.0.0-beta.1-linux-amd64.tar.gz}
```

Comment:
s/v1.0.0-beta.1/%{version}/

Reply:
if this pulls `version` from the version file, this is a non-starter, please see https://github.com/kubernetes-incubator/cri-tools/issues/324.

Reply:
No, this is set automatically to the version that is specified in the Version tag above during [build time](https://docs.fedoraproject.org/quick-docs/en-US/creating-rpm-packages.html#con_rpm-spec-file-overview).

Reply:
I am very happy to get rid of this hard coded version! excellent tip.

Reply:
turns out the version is populated by the version I pass into the pkg_rpm which means this will have an _ which then won't find the file. Overall bummed, but it's all TODO'd.